src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/DefaultValidationMetadataProvider.cs

```diff
@@ -19,6 +19,11 @@ public void GetValidationMetadata([NotNull] ValidationMetadataProviderContext co
             {
                 context.ValidationMetadata.ValiatorMetadata.Add(attribute);
```

Comment:
not this issue and probably not this PR but please correct typo in `ValidatorMetadata` (missing 'd')